UI/Authentication.OidcDemo/Authentication.OidcDemo/Authentication.OidcDemo.Droid/Main.cs

```diff
@@ -28,10 +28,6 @@ public Application(IntPtr javaReference, JniHandleOwnership transfer)
 			: base(() => new App(), javaReference, transfer)
 		{
 			ConfigureUniversalImageLoader();
-
-			ApiExtensibility.Register(
-				typeof(IWebAuthenticationBrokerProvider),
-				_ => new ChromeCustomTabsProvider());
```

Comment:
Was this deprecated ?

Reply:
I restore the code and did the implementation because WebAuthenticationBrokerProvider was changed in the lasest version.